eng/config/PublishData.json

```diff
@@ -28,7 +28,7 @@
       ],
       "vsBranch": "main",
```

Comment:
Does release/dev17.12 need to point to somewhere else now? They're both pointing to main

Reply:
of course it does 🤦